qiskit_optimization/algorithms/optimization_algorithm.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         else:
             if len(x) != len(variables):
                 raise QiskitOptimizationError(
-                    f"Inconsistent size of optimal value and variables. x: size {len(x)} {x}, "
+                    f"Inconsistent size of variables and values. x: size {len(x)} {x}, "
```

Comment:
In this message it looks like you change `optimal value` to `values` (`variable values` more fully above when describing x) and switched the order of that text.r with `variables`, though in the message itself it has values and variables in that order when it prints out the actual quantities that mismatch

Reply:
Thanks. I updated the message as follows. I hope it is comprehensive.
```python
f"Inconsistent size of variable values (x) and variables. x: size {len(x)} {x}, "
f"variables: size {len(variables)} {[v.name for v in variables]}"
```